core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -634,18 +659,7 @@ sealed abstract class IO[+E, +A] extends Serializable { self =>
    * Flattens a nested action with a specified duration.
    */
   final def timeoutFail[E1 >: E](e: E1)(d: Duration): IO[E1, A] =
-    IO.flatten(timeout0[IO[E1, A]](IO.fail(e))(IO.now)(d))
-
-  /**
-   * Returns a new action that executes this one and times the execution.
-   */
-  final def timed: IO[E, (Duration, A)] = timed0(system.nanoTime)
```

Comment:
Is it necessary to remove this? We use it a lot for quick-and-dirty time report logs.

Reply:
no opinion from my side -- just following the description of the ticket

Reply:
No discussion for these on said ticket though... Not sure if we should have one now. Though I'll just passively and un-intrusively indicate my interest in keeping `timed` :)

Reply:
I had no idea there were users. Let's keep them. (Actually I saw someone else using them too.)

Reply:
I'm bringing back `timed` and friends
@jdegoes if you want to object, now is the time :)

Reply:
Nah, I'm good with that. 👍